rust/src/lib.rs

```diff
@@ -136,8 +136,10 @@ macro_rules! blocks {
             blk
     }};
 
-    (@block $a:expr) => { blocks!(@block Const $a); };
+    (@block . $a:expr) => { blocks!(@block Const $a); };
```

Comment:
What's the reason for this `.`?

Maybe a single line of comment to explain it, has it might not be obvious to a new reader (it's not for me at least).

Reply:
Good point.. to be honest it's a workaround still
I wanted to make just regular literals work but I could not
e.g.: `10 (Log)`
but in this macro `(. 10) (Log)` has to be used.
I will implement a `(.)` builtin maybe in mal to allow copy pasting of such scripts.. but not sure we are going to use either.. again WIP but don't wanna loose it in history. 
Will add this text in comments